src/app/containers/ConsentBanner/index.jsx

```diff
@@ -0,0 +1,102 @@
+import React, { Fragment } from 'react';
+import ConsentBanner from '../../components/ConsentBanner';
+import { PlatformContextConsumer } from '../../contexts/PlatformContext';
+import { ServiceContextConsumer } from '../../contexts/ServiceContext';
+
+const parentId = 'consent';
+const promptId = 'consent-prompt';
+
+const ampOnTapHandler = type => ({ on: `tap:${parentId}.${type}` });
```

Comment:
Whilst debugging why the `on` attribute is not being added to the button, I came across this:
```
Note: attributes starting with on are not passed through as an exception because this could become a potential security hole.
```
https://reactjs.org/blog/2017/09/08/dom-attributes-in-react-16.html

It appears that we cannot add the `on` attribute onto React elements.  This `on` attribute is required by AMP - it is used to install event handlers on elements. https://www.ampproject.org/docs/interaction_dynamic/amp-actions-and-events

Reply:
Hey Sareh, did you ever resolve this? Im unable to get this to work
```
const Accept = (message, onClick) => (
  <AmpHelpers.Action events={onClick}>
    {props => <ConsentBannerButton {...props} role="button">{message}</ConsentBannerButton>}
  </AmpHelpers.Action>
);
```
As styled components refuses to set the `on` value